resource_lists/haven/resources.yaml

```diff
@@ -0,0 +1,650 @@
+---
+authors:
+  Ben: "https://benjaminvreeland.com"
+
+index_page_display_name: Haven
+
+game_version: "1.1.302"
+
+note: |
+  Based on
+  https://haven.fandom.com/wiki/Items
+  https://steamdb.info/app/983970/history/
+  https://steamcommunity.com/sharedfiles/filedetails/?id=2310841844
+
+recipe_types:
+  Cooking: Cook {IN_ITEMS} into {OUT_ITEM}
+  Cooking (Camping): Cook {IN_ITEMS} into {OUT_ITEM} (only when camping)
+  Synthesizer: Synthesize {IN_ITEMS} into {OUT_ITEM}
+  Crafting: Craft {IN_ITEMS} into {OUT_ITEM}
+  Enhancement: Use {IN_ITEMS} to enhance {OUT_ITEM}
+  Repair: Use {IN_ITEMS} to repair the Nest ({OUT_ITEM})
+
+resources:
+
+  ##############################################################################
+  _a: {H1: Food} ###############################################################
+  ##############################################################################
+
+  _b: {H2: Kitchen} ############################################################
+
+  Roasted appledews:
```

Comment:
I feel like the other calculators would have named this `Roasted Appledews`. If this pattern is intentional because it matches the pattern in-game then it should stay. I cant find anything conclusive online in my spoiler averse research, and there seem to be some items in this list that do capitalize their secondary words.

Reply:
I was going based on the item list in the wiki, I can check in-game.

Reply:
Looks... inconsistent in-game. I'll either try and match or make it look better.
![image](https://github.com/user-attachments/assets/276ad86f-2db9-4c17-804c-ce380bce1df0)
![image](https://github.com/user-attachments/assets/0d1358ec-a062-4830-bbb7-088fc9b72244)

Reply:
Any way I can help with or validate this. It looks like the recipe for Right Wing is still `Right wing`

Reply:
Based on their own naming scheme `Right Wing` in their UI incorrect, with `Right wing` being consistent both with `Left wing` and other ship parts. We can choose to either match the game exactly or fix their typos, I tend to prefer the latter. 

There is also exactly one item that (in-game) has a period at the end of its name when no others do, and it's not a unique item (similar to the wings there are many examples that are very similar). This also appears to have been a typo so I did not include it in the recipe yaml.